src/constants.py

```diff
@@ -26,3 +26,4 @@
 SYNAPSE_NGINX_PORT = 8080
 SYNAPSE_PORT = 8008
 SYNAPSE_SERVICE_NAME = "synapse"
+SYNAPSE_URL = "http://localhost:8008"
```

Comment:
Do you think we should use the `SYNAPSE_PORT` variable here so that we don't duplicate the configuration? Also, is this synapse URL always on localhost? Or could it be different on production?

Reply:
Didn't notice this when I was aligning my code with main. It seems to be added alongside mjolnir. I'll ask @amandahla about this url

Reply:
moved those constants and deleted the two constant files